### PR TITLE
Adding alt text and fixing a ton of (I mean it) markdown formatting issues

### PR DESCRIPTION
**Pull request by aervnu**
Originally this was to only add alt text, but I found out that markdownlint was reporting multiple formatting issues, so I tried to fix as many of them as possible. I left some ordered lists and inline HTMLs intact (such as tags) since I don't know how to deal with these without making the code look even messier.

I understand the potential of this breaking formatting on some (if not all) pages plus GitHub-related issues, so I welcome all constructive feedback.

### CODE REVIEW

**Comment by aervnu** [comment]
Seems like I reused the commits from an older PR, pretty sure I forgot to sync my repo with the original one. I mean, it's not clean for code maintainers so I can make another entirely new one ig.

**Comment by itsWindows11** [comment]
Rebase the PR branch from upstream `master` branch then re-implement your changes.

File: src/wiki/windows-insiders.md [hunk 1]
```diff
@@ -47,57 +47,57 @@ If this prompt appears more than one time, click **Continue** until it moves on
...
 
 ::: tip Note
 If you don't see all 4 channels, you might be limited by your hardware.
 :::
- 
+
 8. Review the [Microsoft Insider Privacy Statement](https://privacy.microsoft.com/privacystatement) and the [Microsoft Insider Program Agreement](https://insider.windows.com/program-agreement), then click **Continue**.
 
-![](./img/windows-insider/11/agreement-2.png)
+![More agreements smh](./img/windows-insider/11/agreement-2.png)
```
Comment (itsWindows11) [inline src/wiki/windows-insiders.md:58]:
```suggestion
![More agreements](./img/windows-insider/11/agreement-2.png)
```

File: src/wiki/windows-insiders.md [hunk 2]
```diff
@@ -107,19 +107,19 @@ You can choose any account to signup, but your personal Microsoft Account is rec
...
+![Channels](./img/windows-insider/10/channels.png)
 
 ::: tip Note
 If you don't see all 3 channels, you may be limited to certain channels by your hardware.
 :::
 
 9. Read the [Microsoft Insider Privacy Statement](https://privacy.microsoft.com/privacystatement) and the [Microsoft Insider Program Agreement](https://insider.windows.com/program-agreement), then click **Confirm**.
 
-![](./img/windows-insider/10/agreement.png)
+![Agreements again](./img/windows-insider/10/agreement.png)
```
Comment (itsWindows11) [inline src/wiki/windows-insiders.md:118]:
```suggestion
![Agreements](./img/windows-insider/10/agreement.png)
```

File: src/wiki/windows-insiders.md [hunk 3]
```diff
@@ -129,31 +129,31 @@ If you don't want to participate in the Windows Insider Program anymore, you can
...
 
 4. Click on **Leaving the Insider Program**.
 
-![](./img/windows-insider/11/unenroll-immediately.png)
+![Unenroll](./img/windows-insider/11/unenroll-immediately.png)
 
 5. You will be redirected to an official website. Click on *Leave the Program now*.
 
-![](./img/windows-insider/10/leaving-the-program.png)
+![Really leaving](./img/windows-insider/10/leaving-the-program.png)
```
Comment (itsWindows11) [inline src/wiki/windows-insiders.md:148]: Please use more descriptive alts. "Really leaving" isn't really descriptive for users that rely on screenreaders.

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
a

**Comment by FloatingMilkshake** [comment]
Please make sure that alt text is descriptive. One- or two-word descriptions usually aren't descriptive enough.

**Review by FloatingMilkshake — APPROVED** [review]

**Comment by aervnu** [comment]
Really didn't expect you to add them yourself. I appreciate the effort and will remember to utilize alt text more responsibly in the future. I think you're free to merge the PR now, and thanks again for all those fixes!

**Review by Erisa — APPROVED** [review]

**Pull request merged**